src/System.Windows.Forms.Primitives/src/Interop/Richedit/Interop.ENPROTECTED.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal static partial class Richedit
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct ENPROTECTED
+        {
+            public User32.NMHDR nmhdr;
+            public uint msg;
+            public UIntPtr wParam;
+            public IntPtr lParam;
```

Comment:
```cpp
typedef struct _enprotected
{
	NMHDR nmhdr;
	UINT msg;
	WPARAM wParam;
	LPARAM lParam;
	CHARRANGE chrg;
} ENPROTECTED;
```
I _think_ these could be
```suggestion
            public nuint msg;
            public nuint wParam;
            public nint lParam;
```



@JeremyKuhne thoughts?